types/office-js-preview/index.d.ts

```diff
@@ -8864,6 +8864,11 @@ declare namespace Office {
          * [Api set: Mailbox 1.1]
          *
          * **{@link https://docs.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose or Read
+         * 
+         * **Important**: A `recipientType` property value isn't returned by the 
+         * {@link https://docs.microsoft.com/en-us/javascript/api/outlook/office.from?view=outlook-js-preview | Office.context.mailbox.item.from.getAsync} 
+         * and {@link https://docs.microsoft.com/en-us/javascript/api/outlook/office.organizer?view=outlook-js-preview | Office.context.mailbox.item.organizer.getAsync} methods.
+         * The email sender or appointment organizer is always an internal user whose email address is on the Exchange server.
```

Comment:
I don't know if I would say internal. Internal to what?
```suggestion
         * The email sender or appointment organizer is always a user whose email address is on the Exchange server.
```